src/pages/ProviderEServiceDetailsPage/components/ProviderEServiceDetailsTab/ProviderEServiceTechnicalInfoSection/ProviderEServiceTechnicalInfoSection.tsx

```diff
@@ -9,12 +9,15 @@ import { ProviderEServiceThresholdsSection } from './ProviderEServiceThresholdsS
 import { ProviderEServiceUsefulLinksSection } from './ProviderEServiceUsefulLinksSection'
 import { ProviderEServiceDocumentationSection } from './ProviderEServiceDocumentationSection'
 import { useSuspenseQuery } from '@tanstack/react-query'
-import { STAGE } from '@/config/env'
-import type { PagoPAEnvVars } from '@/types/common.types'
+import { FEATURE_FLAG_SIGNALHUB_WHITELIST, SIGNALHUB_WHITELIST } from '@/config/env'
+import { AuthHooks } from '@/api/auth'
 
 export const ProviderEServiceTechnicalInfoSection: React.FC = () => {
-  const signalHubFlagDisabledStage: PagoPAEnvVars['STAGE'][] = ['PROD', 'UAT']
-  const isSignalHubFlagDisabled = signalHubFlagDisabledStage.includes(STAGE) //check on the environment for signal hub flag
+  const producerId = AuthHooks.useJwt().jwt?.organizationId
+  const isSignalHubFlagEnabled = FEATURE_FLAG_SIGNALHUB_WHITELIST
```

Comment:
would standardise the name of the vars as in the other files (both it's fine, one of them)
```suggestion
  const isSignalhubEnabled = FEATURE_FLAG_SIGNALHUB_WHITELIST
```